<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QReads</class>
 <widget class="QWidget" name="QReads">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>704</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>226</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#QReads {
  background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(100,0, 0, 255));
}

QFrame[frameShape=&quot;4&quot;] {
    background-color: #EEEEEE;
    min-height: 10px;
}

QFrame#ImageGroupFrame, QFrame#ToolsGroupFrame {
  border: 6px solid gray;
  background-color: #EEEEEE;
}

QPushButton {
  color: black;
}

QComboBox, QPushButton {
  height: 32px;
}

QDoubleSpinBox, QPushButton, QRadioButton, QComboBox#ZoomComboBox {
  font-weight: bold;
}

QDoubleSpinBox,
QPushButton#SlabButton, QPushButton#ShowReferenceMarkersButton,
QPushButton#ResetReferenceMarkersButton, QComboBox#ZoomComboBox, 
QRadioButton {
  font-size: 13px;
}

QPushButton#ResetWLButton {
  font-size: 14px;
}

QPushButton#CloseApplicationPushButton,
QPushButton#HelpButton
 {
  font-size: 16px;
}

QPushButton#SlabButton,
QPushButton#ShowReferenceMarkersButton,
QPushButton#ResetReferenceMarkersButton
{
  background-color: #EEEEEE;
  border: 1px solid black;
}
QPushButton#SlabButton:checked,
QPushButton#ShowReferenceMarkersButton:checked,
QPushButton#ResetReferenceMarkersButton:checked, QPushButton#SlabButton:hover,
QPushButton#ShowReferenceMarkersButton:hover,
QPushButton#ResetReferenceMarkersButton:hover {
  background-color:#99FFFF;
}
QPushButton#SlabButton:disabled,
QPushButton#ShowReferenceMarkersButton:disabled,
QPushButton#ResetReferenceMarkersButton:disabled {
  background-color: rgb(170, 170, 127);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <widget class="QFrame" name="ImageGroupFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="ImageGroupVerticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="ImageGroupLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../QReads.qrc">:/Icons/SlicerQReads-QREADSImageGroupHeader.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="ImageGroupLine">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ShowReferenceMarkersButton">
        <property name="toolTip">
         <string>Toggle: On = Reference markers Visible | Off = Invisible</string>
        </property>
        <property name="text">
         <string>Show Reference Markers</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ResetReferenceMarkersButton">
        <property name="toolTip">
         <string>Set reference markers back to original orthogonal orientations</string>
        </property>
        <property name="text">
         <string>Reset Reference Markers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SlabButton">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enables Slice Thickness Mode and updates &amp;quot;Reference markers&amp;quot; based on selected thickness.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Set Slice Thickness</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="SlabModeHorizontalLayout">
        <item>
         <widget class="QRadioButton" name="SlabModeMinRadioButton">
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="SlabModeMaxRadioButton">
          <property name="text">
           <string>Max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="SlabModeMeanRadioButton">
          <property name="text">
           <string>Mean</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="ctkSliderWidget" name="SlabThicknessSliderWidget"/>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="3">
         <widget class="QPushButton" name="CTLungWLPresetButton">
          <property name="toolTip">
           <string>Set the window/level for CT Lung</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../QReads.qrc">
            <normaloff>:/Icons/SlicerQReads-CTLung.png</normaloff>:/Icons/SlicerQReads-CTLung.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="ContrastDownButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../QReads.qrc">
            <normaloff>:/Icons/SlicerQReads-ContrastDown.png</normaloff>:/Icons/SlicerQReads-ContrastDown.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="CTBoneWLPresetButton">
          <property name="toolTip">
           <string>Set the window/level for CT Bone</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../QReads.qrc">
            <normaloff>:/Icons/SlicerQReads-CTBone.png</normaloff>:/Icons/SlicerQReads-CTBone.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="CTBodySoftTissueWLPresetButton">
          <property name="toolTip">
           <string>Set the window/level for CT body soft tissue</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../QReads.qrc">
            <normaloff>:/Icons/SlicerQReads-CTBodySoftTissue.png</normaloff>:/Icons/SlicerQReads-CTBodySoftTissue.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="BrightnessUpButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../QReads.qrc">
            <normaloff>:/Icons/SlicerQReads-BrightnessUp.png</normaloff>:/Icons/SlicerQReads-BrightnessUp.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="InverseGrayButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../QReads.qrc">
            <normaloff>:/Icons/SlicerQReads-InverseGray.png</normaloff>:/Icons/SlicerQReads-InverseGray.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="EnableWLButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../QReads.qrc">
            <normaloff>:/Icons/SlicerQReads-WL.png</normaloff>:/Icons/SlicerQReads-WL.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="CTBrainWLPresetButton">
          <property name="toolTip">
           <string>Set the window/level for CT Brain</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../QReads.qrc">
            <normaloff>:/Icons/SlicerQReads-CTHead.png</normaloff>:/Icons/SlicerQReads-CTHead.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="ContrastUpButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../QReads.qrc">
            <normaloff>:/Icons/SlicerQReads-ContrastUp.png</normaloff>:/Icons/SlicerQReads-ContrastUp.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QPushButton" name="ResetWLButton">
          <property name="toolTip">
           <string>Reset window/level to include the entire intensity range (except top/bottom 0.1%, to not let a very thin tail of the intensity distribution to decrease the image contrast too much).</string>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="BrightnessDownButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../QReads.qrc">
            <normaloff>:/Icons/SlicerQReads-BrightnessDown.png</normaloff>:/Icons/SlicerQReads-BrightnessDown.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ZoomLayout">
        <item>
         <widget class="QLabel" name="ZoomLabel">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../QReads.qrc">:/Icons/SlicerQReads-Zoom.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ZoomComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="ToolsGroupFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="ToolsGroupVerticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="ToolsGroupLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../QReads.qrc">:/Icons/SlicerQReads-QREADSToolsGroupHeader.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="ToolsGroupLine">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="toolsGrid">
        <item row="0" column="1">
         <widget class="QPushButton" name="OrientationMarkersButton">
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="RulersButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="DistanceMeasurementButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/Icons/SlicerQReads-DistanceMeasurement.png</normaloff>:/Icons/SlicerQReads-DistanceMeasurement.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>90</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="HelpButton">
     <property name="text">
      <string>Help</string>
     </property>
     <property name="icon">
      <iconset resource="../QReads.qrc">
       <normaloff>:/Icons/SlicerQReads-Help.png</normaloff>:/Icons/SlicerQReads-Help.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="CloseApplicationLayout">
     <item>
      <widget class="QPushButton" name="CloseApplicationPushButton">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../QReads.qrc"/>
 </resources>
 <connections/>
</ui>
